core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -817,6 +818,64 @@ object OracleAnnouncementV0TLV extends TLVFactory[OracleAnnouncementV0TLV] {
   }
 }
 
+sealed trait OracleAttestmentTLV extends TLV {
+  def eventId: NormalizedString
+  def publicKey: SchnorrPublicKey
+  def sigs: Vector[SchnorrDigitalSignature]
+  def outcomes: Vector[NormalizedString]
+}
+
+object OracleAttestmentTLV extends TLVParentFactory[OracleAttestmentTLV] {
+
+  val allFactories: Vector[TLVFactory[OracleAttestmentTLV]] =
+    Vector(OracleAttestmentV0TLV)
+
+  override def typeName: String = "OracleAttestmentTLV"
+}
+
+case class OracleAttestmentV0TLV(
+    eventId: NormalizedString,
+    publicKey: SchnorrPublicKey,
+    sigs: Vector[SchnorrDigitalSignature],
+    outcomes: Vector[NormalizedString])
+    extends OracleAttestmentTLV {
+  require(sigs.nonEmpty, "Cannot have 0 signatures")
+  require(
+    outcomes.size == sigs.size,
+    s"Number of outcomes must match number of signatures, ${outcomes.size} != ${sigs.size}")
+  override val tpe: BigSizeUInt = OracleAttestmentV0TLV.tpe
+
```

Comment:
`tpe` again?

Reply:
It's in the super trait, we have to rename all of them else we have to continue with naming convention for now 

https://github.com/bitcoin-s/bitcoin-s/blob/abc1fdd23fb3480fb3bde10efe1d1f0389211bd7/core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala#L22

Reply:
It takes only a few clicks to rename them all in IntelliJ IDEA.

Reply:
We have different branches that are dependent on this code (adaptor-dlc, all of the branches that are dependent on adaptor-dlc)